plugins/otter-pro/inc/plugins/class-live-search.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Live Search variant.
+ *
+ * @package ThemeIsle\OtterPro\Plugins
+ */
+
+namespace ThemeIsle\OtterPro\Plugins;
+
+/**
+ * Class Live_Search
+ */
+class Live_Search {
+	/**
+	 * The main instance var.
+	 *
+	 * @var Live_Search
+	 */
+	public static $instance = null;
+
+	/**
+	 * Initialize the class
+	 */
+	public function init() {
+		add_filter( 'render_block', array( $this, 'render_blocks' ), 10, 3 );
+	}
+
+	/**
+	 * Block render function for server-side.
+	 *
+	 * @param string $block_content Blocks content.
+	 * @param array  $block Blocks data.
+	 * @return mixed|string
+	 */
+	public function render_blocks( $block_content, $block ) {
+		if ( is_admin() || 'core/search' !== $block['blockName'] || ! isset( $block['attrs']['isLive'] ) ) {
```

Comment:
It's better to change the attribute name to something with otter prefix to avoid any other plugin adding the same attribute.